# Primers and history for 16S amplification

## The regions 

Basically, there are 9 variable regions in the 16S rRNA, and the variablity inside this regions is a temporal map of the different phylogenies of bacteria types. 

Here we can see in an image the regions distributed:

![](https://image.slidesharecdn.com/ampseq-lecture-2013-131113011207-phpapp02/95/amplicon-sequencing-introduction-9-638.jpg?cb=1395905479)

In marine studies, the usual hipervariable regions used have been between the V1 and the V4 (even though intial studies used clonning, getting the complete gene!). 

Along the years of using this gene as a marker to establish the community structure, various attempts have been done in marine science. Let's see some of them. 

## The primers 

### Dataset 
| Name | Region amplified | length amplicon | study | Obvs. |
|-----|-------------------|----------------|--------|-------|
|**8F/1492R**||||The initial one, used by covering the complete region, and resulting in an amplicon of almost 1485 bp. Right now, with the technologies that we use (and Illumina) this is far too big. |
||||||
||||||
|**Bakt_341F/Bakt_805R**|V3-V4|~464|[Herlemann 2011](https://www.ncbi.nlm.nih.gov/pubmed/21472016) |Forward used in BBMO analysis.|
|**358F/806RB**|V3-V4|~448|[Apprill 2015](https://www.int-res.com/articles/ame_oa/a075p129.pdf)|Reverse used in BBMO analysis. Forward not modified ([Caporaso 2011](http://www.pnas.org/content/108/Supplement_1/4516))| 
|**515F/926R**|V4 - V5|~411|[AE Parada 2016](https://www.ncbi.nlm.nih.gov/pubmed/26271760)|Used both in Malaspina, overampli Cyanobacteria|


### Primers !


| Name |Origin  | Sequence|
|------|------|---------|
|**341F**|Herlemann|5'-CCTACGGGNGGCWGCAG|
|**806RB**|Apprill|5'-GGACTACNVGGGTWTCTAAT|
|**515F**|AE Parada|5'-GTGYCAGCMGCCGCGGTAA|
|**926R**|AE Parada|5'-CCGYCAATTYMTTTRAGTTT|

Other info
- : 
- **341F/907R**: This one was used for performing DDGEs . 
- **8F/519R**: Used for 454 pyrosequencing. 
- **515/806R**: EMP. Earth Microbiome Project. 




